viewing_party/party.py

```diff
@@ -1,23 +1,150 @@
 # ------------- WAVE 1 --------------------
 
+from multiprocessing.sharedctypes import Value
+

```

Comment:
Interesting modules, I'm not seeing that you used them in your code though?

Reply:
Thanks Jerica, I'm not sure why that's there. I didn't write it lol